### DISCUSSION TITLE
Support to pycharm

### DISCUSSION THREAD

**Discussion by GabrielJMS** [bernhard-42/vscode-ocp-cad-viewer#88] [Ideas]
Do you think it is possible to do something like this for pycharm?

**Comment by bernhard-42** [comment]
No idea, PyCharm is based on Java and I don't think the frontend is web technology.
Hence it would be a complete new project and codebase

**Comment by bernhard-42** [comment]
btw. you can start the viewer in standalone mode `python -m ocp_vscode`
Then you can use `show` from ocp_vscode in PyCharm, since `show` and the viewer communicate via websockets, which is independent of the editor
